src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,48 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final MovieService movieService = (MovieService) injector
+            .getInstance(MovieService.class);
+    private static final CinemaHallService cinemaHallService = (CinemaHallService) injector
+            .getInstance(CinemaHallService.class);
+    private static final MovieSessionService movieSessionService = (MovieSessionService) injector
+            .getInstance(MovieSessionService.class);
 
+    public static void main(String[] args) {
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHall firstCinemaHall = new CinemaHall(70, "Hall 1");
+        CinemaHall secondCinemaHall = new CinemaHall(85, "Hall 2");
+        cinemaHallService.add(firstCinemaHall);
+        cinemaHallService.add(secondCinemaHall);
+        System.out.println(cinemaHallService.get(1L));
```

Comment:
You could call .getId() hereinafter:
```suggestion
        System.out.println(cinemaHallService.getId());
```

Reply:
In cinemaHallService doesn't have this method getId()

Reply:
Yeah, I'm mixing it up here)
I meant:
```suggestion
        System.out.println(firstCinemaHall.getId());
```